integration-test-common/src/test/java/org/apache/gravitino/integration/test/container/ContainerSuite.java

```diff
@@ -65,6 +65,7 @@ public class ContainerSuite implements Closeable {
   private static volatile KafkaContainer kafkaContainer;
   private static volatile DorisContainer dorisContainer;
   private static volatile HiveContainer kerberosHiveContainer;
+  private static volatile HiveContainer jdbcHiveContainer;
```

Comment:
I think it better to change `jdbcHiveContainer` to `sqlBaseAuthHiveContiner`?

Reply:
Renamed.